orchestrator/ethereum_gravity/src/logic_call.rs

```diff
@@ -193,7 +201,7 @@ fn encode_logic_call_payload(
         sig_arrays.sigs,
         Token::Struct(struct_tokens.to_vec()),
     ];
-    let payload = clarity::abi::encode_call(
+    let payload = gravity_utils::clarity::abi::encode_call(
```

Comment:
why not include just clarity::abi instead ?

Reply:
This is from reexporting. I have just moved the import to the start of the file